lms/djangoapps/courseware/testutils.py

```diff
@@ -90,27 +118,60 @@ def setup_user(self, admin=False, enroll=False, login=False):
         if login:
             self.login()
 
+    def set_block_to_be_requested(self, block_name):
```

Comment:
hmm.. I wasn't expecting a method for this, just a member variable.  But this should be fine.